roles/ssh_hardening/tasks/crypto_hostkeys.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: replace default 2048 bits RSA keypair with 4096 bits keypair
+  openssh_keypair:
+    state: present
+    type: rsa
+    size: 4096
+    path: "{{ ssh_host_keys_dir }}/ssh_host_rsa_key"
+    force: false
+    regenerate: partial_idempotence
+
+# In RHEL and Fedora, the 'ssh_keys' group is the group owner of the host private SSH keys.
+# Since the openssh_keypair module needs to read the key to provide idempotency, we need to set ownership and group based on specific OS vars.
+- name: change host private key ownership, group and permissions
+  file:
+    path: "{{ ssh_host_keys_dir }}/ssh_host_rsa_key"
+    owner: "{{ ssh_host_keys_owner }}"
+    group: "{{ ssh_host_keys_group }}"
+    mode: "0600"
```

Comment:
I found some references, that we need read access for group on RHEL systems on this file, making it `0640` .

see: https://bugzilla.redhat.com/show_bug.cgi?id=1372070

Reply:
You're right. I fixed it in the last commit.